### ISSUE TITLE
Class: RuboCop::Cop::Lint::AmbiguousBlockAssociation wants me to wrap a scope block in parens

### ISSUE THREAD

**Issue by eprislac** [bbatsov/rubocop#4345]
Rubocop incorrectly tells me I should wrap an ActiveRecord scope's block in parens.

## Expected behavior

This shouldn't be calling out for scopes to wrap blocks in parens

## Actual behavior

output from Class: RuboCop::Cop::Lint::AmbiguousBlockAssociation
```ruby
scope :include_thing_true, -> () { where(thing: true) }
 
```
Parenthesize the param -> () { where(thing: true) } to make sure that the block will be associated with the -> method call.

## Steps to reproduce the problem

Create an ActiveRecord scope, run rubocop against it

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.3.1 x86_64-linux)
```

**Comment by Drenmi** [comment]
This has been fixed in `master` as part of #4237. 🙂
